Dockerfile

```diff
@@ -4,6 +4,8 @@ MAINTAINER Steven Mirabito <smirabito@csh.rit.edu>
 
 # Inform about software versions being used inside the builder
 ENV JEKYLL_VERSION=3.2.1
+ENV BUNDLE_PATH=/opt/app-root/bundle
```

Comment:
Worth noting that I chose this completely at random, not sure if there's a better path